admin_app/src/app/content/edit/page.tsx

```diff
@@ -224,6 +240,31 @@ const ContentBox = ({
             placeholder="Add Tags"
           />
         )}
+        filterOptions={(options, params) => {
```

Comment:
I  see that the filtering seems to be an AND, let's do an OR instead as you said? To me showing all contents that have either tag selected in the filter box makes more intutive sense